package.json

```diff
@@ -22,6 +22,7 @@
     "@aws-sdk/client-cognito-identity-provider": "^3.42.0",
     "@emotion/react": "^11.6.0",
     "@emotion/styled": "^11.6.0",
+    "@material-ui/core": "^4.12.3",
```

Comment:
We don't use @material-ui, we use @mui-ui. It's a different versions of same framework